CSV Parse/Program.cs

```diff
@@ -1,18 +1,73 @@
-﻿using System;
+using System;
 using System.IO;
 using System.Text;
 using System.Collections;
 using System.Collections.Generic;
-
+using Syncfusion.XlsIO; 
 using System.Diagnostics;
 
 namespace Program
 {
+    public class ParserEngine
+    {
+        private string _filePath;
+        private string _currentColumn;
+        private string _currentTable;
+
+        public ParserEngine() // Default Constructor
+        {
+            _filePath = "Empty Path";
+            _currentColumn = "No Data";
+            _currentTable = "No Data";
+        }
+
+        public string FilePath // Checks FilePath
```

Comment:
I'm using 'get-set' functions for organization to hide the Exceptions and the test the data before we set it to private variables